.travis.yml

```diff
@@ -4,14 +4,14 @@ node_js:
 - '8'
 matrix:
   include:
-    - env: DOMAIN=https://datahub.io SPECSTOREAPI=https://api.datahub.io
-    - env: DOMAIN=https://testing.datahub.io SPECSTOREAPI=https://api-testing.datahub.io
+    - env: DOMAIN=https://datahub.io API=https://api.datahub.io SPECSTORE=http://api.datahub.io/source
```

Comment:
I did not request this change

Reply:
You did not request it, but I added specstore api, so it is not using it anymore from travis ci env.

Reply:
@Mikanebu I understand that it was necessary to change, but say it in commit message

Reply:
My learning from this PR that I will add every my changes in commit message, so reviewer will not get confused in the future.